js/schedule.js

```diff
@@ -847,10 +906,33 @@ function Schedule(CUSTOM_CONFIG) {
     window.scrollTo(0, 0);
   }
 
+  //Manages state of timeblocks and stores that in localStorage
   schedule.toggleTimeblock = function(timeblockContainer) {
-    timeblockContainer.find(".timeblock-header")
+    timeblockContainer.toggleClass('expanded')
+                      .find(".timeblock-header")
                       .find('.fa').toggleClass('fa-chevron-circle-left fa-chevron-circle-down');
-    schedule.animateBlockToggle(timeblockContainer.find(".sessions-container"));
+    timeblockContainer.find('.sessions-container').slideToggle();
+
+    var timeblockIsOpen = timeblockContainer.hasClass('expanded');
+    var openTimeblocks = JSON.parse(localStorage.getItem('timeblock-states')) || [] ;
```

Comment:
I tend to rely on `Modernizr` for the check: https://github.com/Modernizr/Modernizr/blob/master/feature-detects/storage/localstorage.js since it has more coverage and that we are already utilizing it in our code.

I haven't done much research myself though - I hope Modernizr is pretty up to date with browser specs. (seems like so?)